src/main/java/com/toofifty/easyblastfurnace/methods/GoldHybridMethod.java

```diff
@@ -77,34 +100,36 @@ public MethodStep next(BlastFurnaceState state)
             return waitForBars;
         }
 
-        if (state.getFurnace().has(barItem(), ItemID.GOLD_BAR)) {
-            if (!state.getEquipment().hasIceGlovesEffect()) {
-                return equipIceOrSmithsGloves;
-            }
-            return collectBars;
-        }
+		if (state.getConfig().tickPerfectMethod()) {
+			if (state.getFurnace().hasMoreThanOneInventory(barItem(), ItemID.GOLD_BAR)) {
+				if (!state.getEquipment().hasIceGlovesEffect()) {
+					return equipIceOrSmithsGloves;
+				}
+				return collectBars;
+			}
+		}
+		else if (!state.getConfig().tickPerfectMethod()) {
```

Comment:
Please merge these two conditionals into one like this:

`else if (state.getFurnace().has(barItem(), ItemID.GOLD_BAR)) {`

You don't need to check if `tickPerfectMethod` is disabled as the `else if` covers that.